Cargo.toml

```diff
@@ -31,11 +31,11 @@ serde_json = "1.0.34"
 serde_yaml = "0.8.8"
 hex = "0.3.2"
 
-bitcoin = { version = "0.23", features = [ "use-serde" ] }
-secp256k1 = { version = "0.17", features = [ "recovery" ] }
-tiny-bip39 = "0.6.2"
-lightning-invoice = "0.2.0"
-miniscript = { version = "0.12.0", features = ["compiler"] }
+bitcoin = { version = "0.26.0", features = [ "use-serde" ] }
+secp256k1 = { version = "0.20.1", features = [ "recovery" ] }
+tiny-bip39 = "0.8.0"
```

Comment:
I'm planning to use the bip39 crate here soon (as soon as v1.0.0 releases), so feel free to revert this.